tests/sys/fs/fusefs/ioctl.cc

```diff
@@ -0,0 +1,187 @@
+/*-
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright (c) 2024 CismonX <admin@cismon.net>
```

Comment:
Is CismonX your legal name?  An alias cannot own a copyright.

Reply:
Legally they can. Pseudonyms are explicitly allowed. The project strongly prefers a legal entity, though.